client/src/components/relatedProducts/outfitList.jsx

```diff
@@ -66,24 +66,30 @@ const Outfit = (props) => {
               <AddOutfit
                 handleClick={
                   () => {
-                    let style = productData.styles.results.filter(result => result.style_id === product_style)[0]
+                    let style = productData.styles.results.find(result => result.style_id === product_style)
 
-                    if (list.find(item => item.key === product_id) === undefined) {
+                    if (list.find(item => item.key === product_id+props.addStyle) === undefined) {
 
                       setList((list) => {
 
                         return [...list, <Card
+                        style={props.addStyle}
+                          info={productData.info}
                           picUrls={style.photos.map(photo => photo.url)}
                           category={productData.info.category}
                           name={productData.info.name}
                           price={'$' + style.original_price}
                           salePrice={style.sale_price ?
-                            '$' + style.sale_price : list.length % 2 === 0 ?
-                              '$7327.00' : null}
-                          key={product_id}
-                          button={<FontAwesomeIcon icon={faCircleXmark}
+                            '$' + style.sale_price : null}
+                          key={product_id+props.addStyle}
+                          btnStyle={'x'}
+                          handleXClick = {
+                            ()=>{
+                            console.log('X button clicked from outfitlist' )
+                            setList((list)=> list.filter(item => item.key !== product_id+props.addStyle))
```

Comment:
Console log on line 88

Reply:
will keep for now why building, will only log when clicked.